flink-runtime/src/test/java/org/apache/flink/runtime/rpc/akka/AkkaRpcServiceTest.java

```diff
@@ -135,7 +135,7 @@ public void testGetAddress() {
 	 * @throws ExecutionException
 	 * @throws InterruptedException
 	 */
-	@Test(timeout = 1000)
+	@Test
```

Comment:
It should be ok to remove the timeout, but I think that the build timeout stack traces are not so nice to see which test case actually caused the timeout. The reason for this is that there are multiple test cases being executed in parallel.